src/lua_files/lua/tunnel_ports.lua

```diff
@@ -0,0 +1,29 @@
+if nvrh_mode == "primary" then
```

Comment:
this gos for all these files: the correct scripts are loaded in primary / secondary mode, so these checks in lua can be removed, unless the one feature specifically needs different functionality between primary and secondary modes